recipes/systemc/2.3.3/conanfile.py

```diff
@@ -0,0 +1,112 @@
+        "enable_immediate_self_notifications": [True, False],
+        "enable_pthreads": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "disable_async_updates": False,
+        "disable_copyright_msg": False,
+        "disable_virtual_bind":  False,
+        "enable_assertions": True,
+        "enable_immediate_self_notifications": False,
+        "enable_pthreads": False
+    }
+    generators = "cmake"
+    exports_sources = "patches/**"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+
+        if self.options.shared:
+            del self.options.fPIC
+
+        if self.settings.os == "Macos":
+            raise ConanInvalidConfiguration("Macos build not supported")
+
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("The compilation of SystemC as a "
+                                            "DLL on Windows is currently not "
+                                            "supported")
+
+        if tools.valid_min_cppstd(self, "17"):
+            raise ConanInvalidConfiguration(
+                "C++ Standard %s not supported by SystemC" %
+                self.settings.compiler.cppstd)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("systemc-{}".format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["DISABLE_ASYNC_UPDATES"] = \
+            self.options.disable_async_updates
+        self._cmake.definitions["DISABLE_COPYRIGHT_MESSAGE"] = \
+            self.options.disable_copyright_msg
+        self._cmake.definitions["DISABLE_VIRTUAL_BIND"] = \
+            self.options.disable_virtual_bind
+        self._cmake.definitions["ENABLE_ASSERTIONS"] = \
+            self.options.enable_assertions
+        self._cmake.definitions["ENABLE_IMMEDIATE_SELF_NOTIFICATIONS"] = \
+            self.options.enable_immediate_self_notifications
+        self._cmake.definitions["ENABLE_PTHREADS"] = \
+            self.options.enable_pthreads
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+    def build(self):
+        for patch in self.conan_data["patches"][self.version]:
+            tools.patch(**patch)
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
```

Comment:
Do we need to copy the NOTICE file too? https://github.com/accellera-official/systemc/blob/master/NOTICE

Reply:
If you read the license I would say yes, I will add it